src/utils/material.js

```diff
@@ -10,16 +10,16 @@ module.exports.updateMap = function (shader, data) {
   var el = shader.el;
   var src = data.src;
   var material = shader.material;
-  var materialSystem = el.sceneEl.systems.material;
+  var textureSystem = el.sceneEl.systems.texture;
```

Comment:
I'm really not sure about it. Why not making all the utils systems then? Keeping the 1:1 relationship between component and system is important . 1 component type should be paired with one system or you will be coupling the state of components of different types via those shared systems (scopes). Systems should not be something you can require or there's no difference between a system and a library.